x-pack/qa/sql/src/main/resources/ip.csv-spec

```diff
@@ -0,0 +1,149 @@
+9              |10.0.1.166     |172.27.1.1     
+10             |10.0.1.166     |172.27.1.1      
+;
+
+filterExactMatchIpv6
+SELECT id, client_ip, dest_ip FROM logs WHERE dest_ip = 'fe80::86ba:3bff:fe05:c3f3' ORDER BY id LIMIT 10;
+
+      id       |   client_ip   |         dest_ip         
+---------------+---------------+-------------------------
+19             |10.0.1.222     |fe80::86ba:3bff:fe05:c3f3
+;
+
+
+filterRangeIpv4
+SELECT id, client_ip, dest_ip FROM logs WHERE client_ip BETWEEN '10.0.1.1' AND '10.0.1.200' ORDER BY id LIMIT 10;
+
+      id       |   client_ip   |    dest_ip    
+---------------+---------------+---------------
+1              |10.0.1.166     |172.27.1.1     
+2              |10.0.1.166     |172.27.1.1     
+3              |10.0.1.166     |172.27.1.1     
+4              |10.0.1.166     |172.27.1.1     
+5              |10.0.1.166     |172.27.1.1     
+6              |10.0.1.166     |172.27.1.1     
+7              |10.0.1.166     |172.27.1.1     
+8              |10.0.1.166     |172.27.1.1     
+9              |10.0.1.166     |172.27.1.1     
+10             |10.0.1.166     |172.27.1.1   
+;
+
+filterRangeCIDRIpv4
+SELECT id, client_ip, dest_ip FROM logs WHERE client_ip = '10.0.0.0/16' ORDER BY id LIMIT 10;
+
+      id       |   client_ip   |    dest_ip    
+---------------+---------------+---------------
+1              |10.0.1.166     |172.27.1.1     
+2              |10.0.1.166     |172.27.1.1     
+3              |10.0.1.166     |172.27.1.1     
+4              |10.0.1.166     |172.27.1.1     
+5              |10.0.1.166     |172.27.1.1     
+6              |10.0.1.166     |172.27.1.1     
+7              |10.0.1.166     |172.27.1.1     
+8              |10.0.1.166     |172.27.1.1     
+9              |10.0.1.166     |172.27.1.1     
+10             |10.0.1.166     |172.27.1.1 
+;
+
+filterRangeCIDRIpv6
+SELECT id, client_ip, dest_ip FROM logs WHERE dest_ip = '2001:cafe::/48' ORDER BY id LIMIT 10;
+
+      id       |   client_ip   |           dest_ip            
+---------------+---------------+------------------------------
+20             |10.0.1.199     |2001:cafe::ff07:bdcc:bc59:ff9f
+22             |10.0.1.166     |2001:cafe::ff07:bdcc:bc59:ff9f
+23             |10.0.1.199     |2001:cafe::d46a:9bdc:8126:b00b
+24             |10.0.1.166     |2001:cafe::13e1:16fc:8726:1bf8
+25             |10.0.1.199     |2001:cafe::ff07:bdcc:bc59:ff9f
+26             |10.0.1.166     |2001:cafe::ff07:bdcc:bc59:ff9f
+27             |10.0.1.199     |2001:cafe::13e1:16fc:8726:1bf8
+28             |10.0.1.199     |2001:cafe::ff07:bdcc:bc59:ff9f
+29             |10.0.1.166     |2001:cafe::ff07:bdcc:bc59:ff9f
+30             |10.0.1.199     |2001:cafe::13e1:16fc:8726:1bf8
+;
+
+//
+// below - not supported yet by composite-aggs
+// 
+groupByIp-Ignore
+SELECT client_ip FROM logs GROUP BY client_ip LIMIT 10;
+
+id | client_ip | dest_ip
+1    | 10.0.0.1  | 10.0.0.1
+;
+
+// not supported yet by composite-aggs
```

Comment:
Worth adding a GH issue here, if there is one?

Reply:
Not sure; I've raised https://github.com/elastic/elasticsearch/issues/34766.